.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "prettier.enable" : false
```

Comment:
Did you commit this on purpose?

Reply:
This was done by the json align, but I can revert it, like you suggest for the translation stuff.

Reply:
I'm wrong, it slipped in by mistake